src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/evosuite/SettingsArguments.kt

```diff
@@ -33,12 +33,12 @@ class SettingsArguments(
 
     /**
      * Appends a method parameter to the command.
-     * This makes EvoSuite create goals only for a certain method
-     * of the CUT.
+     * This makes EvoSuite create goals only for a certain method of the CUT.
      *
      * See https://docs.oracle.com/javase/specs/jvms/se11/html/jvms-4.html#jvms-4.3
      *
      * @param method The descriptor of the method we're generating tests for
+     * @return the instance object
      */
     fun forMethod(method: String): SettingsArguments {
```

Comment:
What does this method do, I don't get it. It gets a string and returns `this`?

Reply:
This is not my code, but i think it is a Builder pattern